pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1262,14 +1264,17 @@ private CompletableFuture<Optional<Topic>> createNonPersistentTopic(String topic
                         pulsarStats.recordTopicLoadTimeValue(topic, topicLoadLatencyMs);
                         addTopicToStatsMaps(TopicName.get(topic), nonPersistentTopic);
                         topicFuture.complete(Optional.of(nonPersistentTopic));
-                    }).exceptionally(ex -> {
-                log.warn("Replication check failed. Removing topic from topics list {}, {}", topic, ex.getCause());
-                nonPersistentTopic.stopReplProducers().whenComplete((v, exception) -> {
-                    pulsar.getExecutor().execute(() -> topics.remove(topic, topicFuture));
-                    topicFuture.completeExceptionally(ex);
-                });
-                return null;
-            });
+                    })
+                    .exceptionally(ex -> {
+                        log.warn("Replication check failed. Removing topic from topics list {}, {}",
+                                topic, ex.getCause());
+                        nonPersistentTopic.stopReplProducers()
+                                .whenComplete((v, exception) -> {
+                                    pulsar.getExecutor().execute(() -> topics.remove(topic, topicFuture));
+                                    topicFuture.completeExceptionally(ex);
+                                });
+                        return null;
+                    });
```

Comment:
this looks like a plain formatting change. It's better to avoid such changes.

Reply:
OK, I got it.